gef.py

```diff
@@ -6369,7 +6369,7 @@ def get_insn_size(self, addr):
         next_insn = gef_instruction_n(addr, 2)
         return next_insn.address - cur_insn.address
 
-    @parse_arguments({"address": 0}, {"--nb": 0, })
+    @parse_arguments({"address": "$pc"}, {"--nb": 0, })
```

Comment:
IMO: For consistency it should be `"LOCATION"` instead of `"address"` (which would also involve changing all occurrences of `args.address` to `args.LOCATION`